node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -248,12 +267,16 @@ impl PaymentAdjusterReal {
                     weighed_accounts_affordable_by_transaction_fee,
                 );
 
-            Ok(Either::Left(final_set_before_exhausting_cw_balance))
+            Ok(AccountsByFinalization::Unexhausted(
+                final_set_before_exhausting_cw_balance,
+            ))
         } else {
             let accounts_not_needing_adjustment =
                 convert_collection(weighed_accounts_affordable_by_transaction_fee);
 
-            Ok(Either::Right(accounts_not_needing_adjustment))
+            Ok(AccountsByFinalization::Finalized(
+                accounts_not_needing_adjustment,
+            ))
         }
```

Comment:
Optional (If you're too excited): I think this fn is too long, maybe refactor the if-else block to an outboard method.